packages/widget-chat/src/components/activity-item-post-actions/index.js

```diff
@@ -11,16 +11,31 @@ export default function ActivityItemPostActions(props) {
     onDelete(id);
   }
 
+  const actions = [];
+  if (props.isSelf) {
+    actions.push(
+      {
+        handleOnClick: handleOnDelete,
+        title: `Delete this message`,
+        type: ICON_TYPE_DELETE
+      });
```

Comment:
This feels like an action constructor in a presentational object, and maybe needs to be a level higher. Does this component need to know if it's self or not, or can it just take an action when passed to it?
I think there's definitely a fine line between handling too much in parent components or too much in low level presentational components, and I'm not sure what is best. Any thoughts @ianwremmel or @Josh-Dykstra?